scripts/change-log-generator.js

```diff
@@ -80,23 +63,14 @@ function getReleaseBranch() {
  */
 function getReleaseBranches() {
   return shell
-    .exec(GIT_BRANCH, { silent: true })
+    .exec("git branch -r -l -s='-creatordate' 'origin/release/v*'", {
+      silent: true
+    })
     .replace(/\n/g, ',')
     .split(',')
     .map(Function.prototype.call, String.prototype.trim);
 }
 
-function getPreviousReleaseBranch(releaseBranch) {
```

Comment:
I think it will be better to diff against master going forward instead of the previous release branch.